jobclient/java/src/main/java/com/twosigma/cook/jobclient/Job.java

```diff
@@ -1084,6 +1086,7 @@ public static JSONObject jsonizeJob(Job job)
         if (job.getDisk().shouldIncludeInJSON()) {
             object.put("disk", job.getDisk().toJSONObject());
         }
+        object.put("status", job.getStatus().name());
```

Comment:
This doesn't seem right -- we don't want to send status on job submissions, right?

Reply:
I'm ok with removing this but you would need to remove the requirement that status is set when parsing from JSON -> Job.

Otherwise you would have an issue where you have a `Job job`, you initialize `JSONObject json = jsonizeJob(job)`, when you call `parseFromJson(json)`, you'll run into [an issue parsing 'status' from the JSON](https://github.com/twosigma/Cook/blob/master/jobclient/java/src/main/java/com/twosigma/cook/jobclient/Job.java#L1204).